src/Moq/Properties/Resources.resx

```diff
@@ -348,4 +348,7 @@ Expected invocation on the mock once, but was {4} times: {1}</value>
 		<value>The following invocations on mock '{0}' were not verified:
 {1}</value>
 	</data>
-</root>
\ No newline at end of file
+	<data name="CallBaseUsedOnDelegateException" xml:space="preserve">
+    <value>CallBase cannot be used to mock Delegate types.</value>
```

Comment:
I'd suggest a slight rephrasing here:

```diff
-CallBase cannot be used to mock Delegate types.
+CallBase cannot be used with Delegate mocks.
```

(The reason being that `CallBase` isn't the party that mocks something; the delegate mock exists already by the time `CallBase` is being set.)